### ISSUE TITLE
Exception: Cannot read property 'replacements' of undefined

### ISSUE THREAD

**Issue by vuhrmeister** [dubvfan87/meteor-emoticons#2]
I can't use this package. I just installed it and used the helper in my template. I only get this exception logged to console:

```
Exception in template helper: TypeError: Cannot read property 'replacements' of undefined
```

I did not set any custom emoticons, I'm just using out-of-the-box installation.

**Comment by mmoyles87** [comment]
Sorry, just saw this today. I've fixed it in version 0.0.4

**Comment by vuhrmeister** [comment]
Cool thx
